pkg/multicluster/razee/razee.go

```diff
@@ -280,54 +18,21 @@ func (t *RazeeGithubApiRoundTripper) RoundTrip(request *http.Request) (*http.Res
 	return http.DefaultTransport.RoundTrip(request)
 }
 
-//nolint:golint,unused
-type RazeeLocalRoundTripper struct {
-	url      string
-	login    string
-	password string
-	token    string
-}
+const TokenValidityDuration = 40 * time.Minute
```

Comment:
Did you check if the token has expiration (`exp`) within it? If so we can remove this hardcoding in a future PR.

Reply:
The token is unfortunately only a token. 40 minutes is an estimation. I checked the code and in Razee it defines `240m` as validity period. We can set it to 200 min to let it be valid for more but I would not stress it and set it to 240.

Reply:
Let's handle the details of this in the external PR to the satcon library https://github.com/IBM/satcon-client-go/pull/24